mipdb/commands.py

```diff
@@ -26,14 +26,17 @@
 from mipdb.usecases import UntagDataset
 from mipdb.usecases import ValidateDataset
 
-_ip_port_options = [
+_db_configs_options = [
     cl.option("--ip", "ip", required=False, help="The ip of the database"),
     cl.option("--port", "port", required=False, help="The port of the database"),
+    cl.option(
```

Comment:
The `password` option should be read from an environment variable by default.

Maybe do this for the rest as well.